x-pack/plugins/ml/server/routes/schemas/data_analytics_schema.ts

```diff
@@ -10,6 +10,7 @@ import { runtimeMappingsSchema } from './runtime_mappings_schema';
 
 export const dataAnalyticsJobConfigSchema = schema.object({
   description: schema.maybe(schema.string()),
+  _meta: schema.maybe(schema.any()),
```

Comment:
```suggestion
  _meta: schema.maybe(schema.object()),
```

Reply:
This one I left as `any` because `schema.object` requires the properties to be defined and `_meta` can take whatever properties we want to add. So this could be problematic  as it's volatile and for `_meta` properties that get added automatically on job creation, etc.

Reply:
So in the `[PR](https://github.com/elastic/elasticsearch/pull/94529)` adding it in it's described as
`The _meta field stores an arbitrary key-value map. Keys are strings. Values are arbitrary objects
(possibly also maps).`

I don't believe schema let's you specify a key type without knowing what the key is. Happy to add it in if I'm wrong about that.

Reply:
The TS spec now defines it as an optional record like `_meta: undefined | Record<string, any>;` (in other places `_meta: undefined | Record<string, unknown>;`) but this one just as an optional `any`, maybe it can be updated to also be like the `Record`?